cl/sentinel/discovery.go

```diff
@@ -30,14 +30,21 @@ import (
 	"github.com/erigontech/erigon/cl/clparams"
 	"github.com/erigontech/erigon/p2p/enode"
 	"github.com/erigontech/erigon/p2p/enr"
+	"golang.org/x/sync/semaphore"
 )
 
-const peerSubnetTarget = 4
+const (
+	peerSubnetTarget                 = 4
+	GoRoutinesOpeningPeerConnections = 4
```

Comment:
lowercase change this to `goRoutinesOpeningPeerConnections`

Reply:
Fixed, but as I know const variable naming convention is to start with capital letter. Should I always start with lowercase or is it just in this file because of another const name?